doc/Motr_Terminology_Guide.rst

```diff
@@ -0,0 +1,161 @@
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Storage object (stob)                                   | A storage object (stob) is a container for unstructured data, accessible through m0\_stob interface. Back-End (BE) uses stobs to store meta-data on persistent store. BE accesses persistent store only through m0\_stob interface and assumes that every completed stob write survives any node failure. It is up to a stob implementation to guarantee this                                                                                                                                                                                                                          |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Striping pattern                                        | A striping layout belongs to a striping pattern (N+K)/G. If it stores the K parity unit with the redundancy information for every N data unit and the units are stored in G containers. Typically G is equal to the number of storage devices in the pool. Where G is not important or clear from the context, one talks about N+K striping pattern (which coincides with the standard RAID terminology).                                                                                                                                                                              |
```

Comment:
A striping layout belongs to a striping pattern (N+K)/P. If it stores the K parity units with the redundancy information for every N data units and the units are stored in P containers. Typically P is equal to the number of storage devices in the pool. Where P is not important or clear from the context, one talks about N+K striping pattern (which coincides with the standard RAID terminology)